### ISSUE TITLE
There is no easy way to query the instance version while still supporting 1.0

### ISSUE THREAD

**Issue by koute** [MaikKlein/ash#195]
Vulkan has the `vkEnumerateInstanceVersion` function which can be used to query the maximum API version an instance can be created for on a system, however it is only available since 1.1, which is problematic if you actually want to check if 1.1 is supported...

`ash` currently exposes the `vkEnumerateInstanceVersion` as `ash::vk::EntryFnV1_1::enumerate_instance_version` which panics when `vkEnumerateInstanceVersion` is missing, so it's somewhat useless as you can't call it unless you first manually check with `get_instance_proc_addr` that `vkEnumerateInstanceVersion` is actually available.

Also, it looks like with `ash` 0.28 the `enumerate_instance_version` cannot be actually called? If I do this:

```rust
use ash::version::EntryV1_1;
entry.enumerate_instance_version();
```

I get:

```
error[E0599]: no method named `enumerate_instance_version` found for type `ash::entry::EntryCustom<std::sync::Arc<shared_library::dynamic_library::DynamicLibrary>>` in the current scope
```

Calling e.g. `enumerate_instance_layer_properties` on the same object works as expected.

**Comment by aloucks** [comment]
This is addressed in #194  

https://github.com/MaikKlein/ash/pull/194/files#diff-ea4f4f59e03601289515af74c8c3e200R274

**Comment by aloucks** [comment]
@MaikKlein  I think this can be closed now.
